Classes/DataProcessing/ContainerProcessor.php

```diff
@@ -35,10 +37,20 @@ class ContainerProcessor implements DataProcessorInterface
      */
     protected $contentDataProcessor;
 
-    public function __construct(ContainerFactory $containerFactory, ContentDataProcessor $contentDataProcessor)
+    /**
+     * @var LanguageAspect
+     */
+    protected $languageAspect;
+
+    public function __construct(ContainerFactory $containerFactory, ContentDataProcessor $contentDataProcessor, Context $context = null)
```

Comment:
I suggest fetching the Context from TSFE->getContext() as Context is not a stateless information and might be changed if you Dataprocessor has been initialized early enough (this is a similar situation to PageRepository in Core).

cObj->getTypoScriptFrontendController->getContext() should work IMHO.